### PR TITLE
Fixed --HEAD build

### PR DESCRIPTION
**Pull request by dqh-au**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/vice.rb [hunk 1]
```diff
@@ -39,20 +39,19 @@ def install
...
       "--disable-arch",
-      "--disable-bundle",
       "--enable-external-ffmpeg",
     ]
 
     if build.head?
       configure_flags << "--enable-native-gtk3ui"
-      configure_flags << "--disable-hwscale"
     else
+      configure_flags << "--disable-bundle"
```
Comment (fxcoudert) [inline Formula/vice.rb]: Please explain the reasons behind the changes.

File: Formula/vice.rb [hunk 2]
```diff
@@ -39,20 +39,19 @@ def install
...
       configure_flags << "--enable-sdlui2"
     end
 
     system "./autogen.sh" if build.head?
     system "./configure", *configure_flags
-    system "make", "install"
+    system "make", "-j", "install"
```
Comment (fxcoudert) [inline Formula/vice.rb]: No, this is taken care of automatically behind the scenes.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/vice.rb (hunk 1 above)
Comment (dqh-github) [reply Formula/vice.rb]: --disable-bundle was required in vice 3.3 to allow a 'make install' - required for brew. The HEAD codebase no longer requires or supports this flag.

--disable-hwscale was previously required for macOS gtk HEAD builds as openGL wasn't being used correctly on macOS.

Comment (dqh-github) [reply Formula/vice.rb]: ok, that change is made.

File: Formula/vice.rb (hunk 2 above)
Comment (dqh-github) [reply Formula/vice.rb]: ok, that change is made

**Comment by fxcoudert** [comment]
Thank you @dqh-github

**Pull request merged**